<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
   <property name="windowTitle">
    <string>Clanker Factory Command</string>
   </property>
    <widget class="QWidget" name="centralwidget">
     <layout class="QVBoxLayout" name="centralLayout">
        <item>
         <layout class="QHBoxLayout" name="statusRowLayout">
            <item>
             <widget class="QGroupBox" name="factoryGroupBox">
                <property name="title">
                 <string>Factory Status</string>
                </property>
                <layout class="QVBoxLayout" name="factoryStatusLayout">
                 <item>
                    <layout class="QHBoxLayout" name="factoryHealthLayout">
                     <item>
                        <widget class="QLabel" name="factoryHealthLabel">
                         <property name="text">
                            <string>Health</string>
                         </property>
                        </widget>
                     </item>
                     <item>
                        <widget class="QProgressBar" name="factoryHealthBar">
                         <property name="maximum">
                            <number>500</number>
                         </property>
                         <property name="value">
                            <number>500</number>
                         </property>
                        </widget>
                     </item>
                    </layout>
                 </item>
                 <item>
                    <widget class="QLabel" name="resourcesLabel">
                     <property name="text">
                        <string>Resources: 0</string>
                     </property>
                    </widget>
                 </item>
                 <item>
                    <widget class="QLabel" name="batteryLabel">
                     <property name="text">
                        <string>Batteries: 0</string>
                     </property>
                    </widget>
                 </item>
                </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="enemyGroupBox">
                <property name="title">
                 <string>Threat Monitor</string>
                </property>
                <layout class="QVBoxLayout" name="enemyLayout">
                 <item>
                    <widget class="QLabel" name="enemyInfoLabel">
                     <property name="text">
                        <string>Spotted enemies</string>
                     </property>
                     <property name="alignment">
                        <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                 </item>
                 <item>
                    <widget class="QLabel" name="enemyCountLabel">
                     <property name="font">
                        <font>
                         <pointsize>18</pointsize>
                         <weight>75</weight>
                         <bold>true</bold>
                        </font>
                     </property>
                     <property name="text">
                        <string>Enemies: 0</string>
                     </property>
                     <property name="alignment">
                        <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                 </item>
                 <item>
                    <widget class="QPushButton" name="damageButton">
                     <property name="text">
                        <string>Simulate Incoming Damage</string>
                     </property>
                    </widget>
                 </item>
                </layout>
             </widget>
            </item>
         </layout>
        </item>
        <item>
            <widget class="QGroupBox" name="clankerGroupBox">
             <property name="title">
                <string>Clanker Squads</string>
             </property>
             <layout class="QVBoxLayout" name="clankerTableLayout">
              <item>
               <widget class="QLabel" name="clankerTableHintLabel">
                <property name="text">
                 <string>Counts update live as you build or lose units.</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="clankerTableWidget">
                <property name="alternatingRowColors">
                 <bool>true</bool>
                </property>
                <property name="editTriggers">
                 <set>QAbstractItemView::NoEditTriggers</set>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::NoSelection</enum>
                </property>
                <property name="showGrid">
                 <bool>false</bool>
                </property>
                <property name="focusPolicy">
                 <enum>Qt::NoFocus</enum>
                </property>
                <property name="rowCount">
                 <number>3</number>
                </property>
                <property name="columnCount">
                 <number>3</number>
                </property>
                <column>
                 <property name="text">
                  <string>Squad</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Active Units</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Avg Energy</string>
                 </property>
                </column>
                <row>
                 <property name="text">
                  <string>Workers</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>Scouts</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>Defenders</string>
                 </property>
                </row>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="unitTableHintLabel">
                <property name="text">
                 <string>Individual squads appear below with live energy and status.</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="clankerDetailTableWidget">
                <property name="alternatingRowColors">
                 <bool>true</bool>
                </property>
                <property name="editTriggers">
                 <set>QAbstractItemView::NoEditTriggers</set>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::NoSelection</enum>
                </property>
                <property name="showGrid">
                 <bool>false</bool>
                </property>
                <property name="focusPolicy">
                 <enum>Qt::NoFocus</enum>
                </property>
                <property name="rowCount">
                 <number>0</number>
                </property>
                <property name="columnCount">
                 <number>5</number>
                </property>
                <column>
                 <property name="text">
                  <string>ID</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Name</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Class</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Energy</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Status</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="controlsAndLogLayout">
            <item>
             <widget class="QGroupBox" name="controlsGroupBox">
                <property name="title">
                 <string>Factory Controls</string>
                </property>
                <layout class="QGridLayout" name="controlsGridLayout">
                 <item row="0" column="0">
                    <widget class="QLabel" name="clankerTypeLabel">
                     <property name="text">
                        <string>Choose clanker type</string>
                     </property>
                    </widget>
                 </item>
                 <item row="1" column="0">
                    <widget class="QComboBox" name="clankerTypeComboBox">
                     <item>
                        <property name="text">
                         <string>Worker</string>
                        </property>
                     </item>
                     <item>
                        <property name="text">
                         <string>Scout</string>
                        </property>
                     </item>
                     <item>
                        <property name="text">
                         <string>Defender</string>
                        </property>
                     </item>
                    </widget>
                 </item>
                 <item row="2" column="0">
                    <widget class="QPushButton" name="produceButton">
                     <property name="text">
                        <string>Produce Selected Clanker</string>
                     </property>
                    </widget>
                 </item>
                 <item row="3" column="0">
                    <widget class="QPushButton" name="produceBatteryButton">
                     <property name="text">
                        <string>Produce Battery</string>
                     </property>
                    </widget>
                 </item>
                 <item row="4" column="0">
                    <widget class="QPushButton" name="pauseButton">
                     <property name="text">
                        <string>Pause</string>
                     </property>
                    </widget>
                 </item>
                </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="logGroupBox">
                <property name="title">
                 <string>Mission Log</string>
                </property>
                <layout class="QVBoxLayout" name="logLayout">
                 <item>
                    <widget class="QLabel" name="logHintLabel">
                     <property name="text">
                        <string>Latest events</string>
                     </property>
                    </widget>
                 </item>
                 <item>
                   <widget class="QTextEdit" name="logTextEdit">
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                     <property name="placeholderText">
                        <string>Production updates, combat results, and warnings appear here.</string>
                     </property>
                    </widget>
                 </item>
                </layout>
             </widget>
            </item>
         </layout>
        </item>
     </layout>
    </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
